script/run_tests_in_docker

```diff
@@ -0,0 +1,16 @@
+#!/bin/sh
+
```

Comment:
Please  move the logic to the rake file. And modify `script/test` instead. Something like this.

~~~bash
#!/usr/bin/env bash

argument=$1
shift

if [ "$argument" == 'docker' ]; then
  bundle exec rake test $*
  echo bundle exec rake test $*
else
  bundle exec rake test:docker $*
fi
~~~